insights/parsers/ip.py

```diff
@@ -280,7 +280,7 @@ class RouteDevices(CommandParser):
     * ``netmask`` - the CIDR notation of the network as a string (e.g. '24')
     * ``table`` - the routing table, or None if not given.
 
-    Destinations given as **broadcast**, **throw**, **local**,
+    Destinations given as **broadcast**, **throw**,
     **unreachable**, **prohibit**, **blackhole** and **nat** are discarded.
 
     Destinations given as **unicast**, **multicast**
```

Comment:
@vishwanathjadhav this docstring line is incomplete, looks like its missing the rest of the sentence.

Reply:
@bfahr - no, the line is complete, just required to adjust the line. The changes are done.

Reply:
@vishwanathjadhav - I think what @bfahr pointed out is the line-286, but not the line-283, would you please to help complete the line-286?

Reply:
@xiangce , @bfahr - Thanks, I have update the doc